src/components/containers/ConnectedPositionTable/ConnectedPositionTable.tsx

```diff
@@ -121,6 +122,14 @@ const ConnectedPositionTable: React.FunctionComponent<ConnectedAMMTableProps> =
 
     const spData = portfolioData.info[positionToSettle.position.id];
 
+    let netWithdraw = undefined;
+    if (agent === Agents.LIQUIDITY_PROVIDER) {
+      netWithdraw = !isUndefined(spData?.fees) && !isUndefined(spData?.settlementCashflow) ? spData?.margin + spData?.fees + spData?.settlementCashflow : undefined;
```

Comment:
according to https://lodash.com/docs/4.17.15#isUndefined isUndefined(null) returns false. This would mean if spData?.settlementCashflow and spData?.fees are null we can end up in null + null scenario (gives NaN). Not sure if this is the desired outcome, perhaps you need to reflect based on this comment :D

Reply:
fair point, I created a notion ticket `UI isUndefined() refactoring` for this refactoring